decidim-proposals/app/commands/decidim/proposals/admin/create_note_proposal.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      # A command with all the business logic when an admin creates a private note proposal.
+      class CreateNoteProposal < Rectify::Command
```

Comment:
This should be `CreateProposalNote` right?

Reply:
You are right!